### ISSUE TITLE
How to support php 7.1

### ISSUE THREAD

**Issue by Cuihongsen** [robregonm/yii2-pdf#22]
How to support php 7.1

**Comment by robregonm** [comment]
Theoretically, it is already supported. If you are experiencing issues, feel free to report here (if it is related to this library) or report to mpdf repo (in case it is related to mpdf directly)

**Comment by zoran-grujic** [comment]
No, PHP 7 is not actually supported. It is required to **update mpdf** first, but then **yii2-pdf is not** compatible **with mpdf v7**.  To make it compatible one need to change class name from mPDF to Mpdf and Mpdf accepts single input parameter that is actually array of mPDF parameters. See post:

https://harrybailey.com/2018/08/yii-1-php-7-1-yii-pdf-and-mpdf/

I would like to update in a clean way **yii2-pdf** and **mpdf v7**. Is there an easy way to do it? Shall I fork **yii2-pdf** and make it work for **mpdf v7**?

**Comment by robregonm** [comment]
Feel free to submit a PR, or to become a contributor :-)

**Comment by zoran-grujic** [comment]
I am not big git master, but I think I managed to send two pull requests with changes that might do the job. And you can mark it as new version after merge...

**Comment by Julian-B90** [comment]
@robregonm i have made a PR for this issue #29
